.github/workflows/build-and-push.yml

```
@@ -0,0 +1,155 @@
+name: build and push
+
+on:
+  push:
+
+jobs:
+  npm-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - run: npm install
```

We should use [`npm ci`](https://docs.npmjs.com/cli/v6/commands/npm-ci#description) instead of `npm install`. [Travis was doing that automatically](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#npm-ci-support)
```suggestion
      - run: npm ci
```

---

Oh, that makes a lot of sense. Fixing now